VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -57,60 +68,63 @@ public void render(BaseViewHolder tag, DiscussionThread discussionThread) {
                 }
             } else {
                 icon = FontAwesomeIcons.fa_comments;
-                iconColor = edx_grayscale_neutral_light;
+                iconColor = (discussionThread.isRead() ? edx_grayscale_neutral_dark : edx_brand_primary_base);
             }
             holder.discussionPostTypeIcon.setIcon(icon);
             holder.discussionPostTypeIcon.setIconColor(iconColor);
         }
 
         {
-            String threadTitle = discussionThread.getTitle();
-            holder.discussionPostTitle.setText(threadTitle);
+            final SpannableString threadTitle = new SpannableString(discussionThread.getTitle());
+            if (!discussionThread.isRead()) {
+                CalligraphyTypefaceSpan boldSpan = new CalligraphyTypefaceSpan(openSansSemiBoldFont);
+                threadTitle.setSpan(boldSpan, 0, threadTitle.length(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+                holder.discussionPostTitle.setText(threadTitle);
+            } else {
+                // Calling toString will truncate the spannable properties applied (if any)
+                holder.discussionPostTitle.setText(threadTitle.toString());
+            }
         }
 
         holder.discussionPostClosedIcon.setVisibility(discussionThread.isClosed() ? View.VISIBLE : View.GONE);
         holder.discussionPostPinIcon.setVisibility(discussionThread.isPinned() ? View.VISIBLE : View.GONE);
         holder.discussionPostFollowIcon.setVisibility(discussionThread.isFollowing() ? View.VISIBLE : View.GONE);
 
         {
-            String authorLabel = discussionThread.getAuthorLabel();
-            if (authorLabel != null) {
-                holder.discussionPostAuthor.setVisibility(View.VISIBLE);
-                holder.discussionPostAuthor.setText(getContext().getString(R.string.discussion_priviledged_author_attribution, authorLabel));
-
+            final int commentCount = discussionThread.getCommentCount();
+            if (commentCount == 0) {
+                holder.discussionPostRepliesTextView.setVisibility(View.GONE);
             } else {
-                holder.discussionPostAuthor.setVisibility(View.GONE);
+                CharSequence totalReplies = ResourceUtil.getFormattedString(
+                        getContext().getResources(), R.string.discussion_post_total_replies,
+                        "total_replies", commentCount > 99 ? "99+" : String.valueOf(commentCount));
```

Comment:
No, it means 99 or greater: https://math.stackexchange.com/questions/480123/plus-sign-after-a-number-means